config/application.rb

```diff
@@ -28,5 +28,10 @@ class Application < Rails::Application
     # the framework and any gems in your application.
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    #Configure cookie store for middlewear
+    config.session_store :cookie_store, key: '_interslice_session'
+    config.middleware.use ActionDispatch::Cookies # Required for all session management
+    config.middleware.use ActionDispatch::Session::CookieStore, config.session_options
```

Comment:
Isn't this usually provided by default? If we need it then I'd prefer to see this configured in its own initializer?

Reply:
No according to the Omniauth documentation at https://github.com/omniauth/omniauth#integrating-omniauth-into-your-rails-api You have to configure the middle wear to use cookies.

Reply:
That's for a Rails API, which typically don't enable cookies. We're not implementing one so we've already got them enabled.